lib/commands/context.js

```diff
@@ -117,12 +115,21 @@ commands.mobileGetContexts = async function mobileGetContexts () {
   return await webviewHelpers.getWebViewsMapping(this.adb, opts);
 };
 
-helpers.switchContext = async function switchContext (name) {
+helpers.assignContexts = function assignContexts (webviewsMapping) {
+  const webviews = this.isChromeSession ?
+    [CHROMIUM_WIN] :
+    webviewHelpers.parseWebviewNames(webviewsMapping, this.opts.ensureWebviewsHavePages);
```

Comment:
I assume the second argument should be a dictionary

Reply:
I'm sorry for that. Fixed as https://github.com/appium/appium-android-driver/pull/699/commits/240e851880ebb0716275400a4c217432bb8a40a9